web_src/js/components/RepoActionView.vue

```diff
@@ -250,21 +286,37 @@ const sfc = {
       lineNumber.textContent = line.index;
       div.append(lineNumber);
 
-      // TODO: Support displaying time optionally
-
-      const logMessage = document.createElement('div');
+      // for "Show timestamps"
+      const logTimeStamp = document.createElement('span');
+      logTimeStamp.className = 'log-time-stamp';
+      const date = new Date(parseFloat(line.timestamp * 1000));
+      const timeStamp = date.toLocaleString(getCurrentLocale(), {year: 'numeric', month: 'short', day: '2-digit', weekday: 'short', hour: '2-digit', hour12: false, minute: '2-digit', second: '2-digit', timeZone: 'UTC', timeZoneName: 'short'});
+      logTimeStamp.innerHTML = timeStamp;
+      toggleElem(logTimeStamp, this.timeVisible.stamp);
+      // for "Show log duration"
+      const logTimeDuration = document.createElement('span');
+      logTimeDuration.className = 'log-time-duration';
+      const duration = Math.floor(parseFloat(line.timestamp) - parseFloat(startTime));
+      logTimeDuration.innerHTML = `${duration}s`;
```

Comment:
`.textContent`?

Reply:
Changed in https://github.com/go-gitea/gitea/pull/24876/commits/524275e7217cdbcd60d027d56a8c0a6ed2a1e351